File: furnitureshop-backend/src/main/java/domain/shop/Manufacturer.java
package domain.shop;

/**
 * A object that represents manufacturer of product
 */
public class Manufacturer {

    private Long id;
    private String title;
    private Requisite requisite;

    public Manufacturer() {
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public Requisite getRequisite() {
        return requisite;
    }

    public void setRequisite(Requisite requisite) {
        this.requisite = requisite;
    }
}

File: furnitureshop-frontend/src/domain/product/manufacturer.ts
import {Requisite} from '../shop/requisite';

export class Manufacturer {

  private _id: number;
  private _title: string;


  constructor(title: string) {
    this._title = title;
  }

  get title(): string {
    return this._title;
  }

  set title(value: string) {
    this._title = value;
  }

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }
}
